prow/cmd/deck/main.go

```diff
@@ -912,7 +912,7 @@ func renderSpyglass(sg *spyglass.Spyglass, cfg config.Getter, src string, o opti
 	src = strings.TrimSuffix(src, "/")
 	realPath, err := sg.ResolveSymlink(src)
 	if err != nil {
-		return "", fmt.Errorf("error when resolving real path: %v", err)
+		return "", fmt.Errorf("error when resolving real path '%v': %v", src, err)
```

Comment:
Consider [wrapping](https://blog.golang.org/go1.13-errors#TOC_3.3.) errors in the future with `%w`

src is a string so I'd use `%s` or `%q`

Reply:
Ack'd